.vib/appsmith/cypress/cypress/integration/appsmith_spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from '../support/utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.get('header div[class*="profile-menu"]').click();
+  cy.contains('Sign Out').click();
+  cy.contains('sign in');
+});
+
+it('allows to create a new project', () => {
+  cy.login();
+  cy.get('button[class*="createnew"]').click();
+  // We don't need a name as it already adds a random name
+  cy.contains('Widgets');
+  // There is a pop-up (when there is a new version available) that takes some
+  // animation to appear, but when it does, it blocks a button. We need to wait
+  // a few seconds to let it appear and then remove it so we can continue
+  cy.wait(2000)
+  cy.get('body').then(($body) => {
+    // Close the pop-up if appears
+    if ($body.find('[class*="toast-action"]').is(':visible')) {
+      cy.get('[class*="toast-action"]').click();
+    }
+  });
```

Comment:
Statically waiting for a prefixed period of time is considered to be [an anti-pattern](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting), and should be avoided. 

In this case, if the pop-up is blocking the next selectable item (and prevents to click it), you may use the `force: true` option instead, which ignores if an element is covered.